core/src/main/java/com/opensymphony/xwork2/spring/SpringUtils.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2017 The Apache Software Foundation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.opensymphony.xwork2.spring;
+
+import com.opensymphony.xwork2.util.ClassLoaderUtil;
+import org.apache.commons.lang3.reflect.MethodUtils;
+
+import java.lang.reflect.Proxy;
+
+/**
+ * <code>SpringUtils</code>
+ * <p>
+ * Various utility methods dealing with spring framework
+ * </p>
+ *
+ */
+public class SpringUtils {
```

Comment:
Why someone may need to override such specific methods?

As all methods could be static, I thought normally it should be a static utils class rather than a bean object.